recipes-extended/player-interface/player-interface_git.bb

```diff
@@ -0,0 +1,69 @@
+SUMMARY = "Middleware Player Interface layer for Player"
+DESCRIPTION = "This layer provides the Player Firebolt Interface library for Player integration."
+LICENSE = "CLOSED"
```

Comment:
LICENSE must not be closed - SRC_URI is an opensourced repo.  Please use "Apache-2.0".